benchmark/profiling/prepare/instances/SerialiseInstances.hs

```diff
@@ -11,11 +11,17 @@ import           Clash.Annotations.Primitive                  (PrimitiveGuard)
 import qualified Clash.Annotations.BitRepresentation.Internal as CP
 
 import qualified Clash.Primitives.Types                       as CL
-import qualified Clash.Netlist.Types                          as CL (BlackBox)
+import qualified Clash.Netlist.Types                          as CL (BlackBox, TopEntityT)
 import qualified Clash.Netlist.BlackBox.Types                 as CL
 
 import qualified Data.HashMap.Strict                          as HM
 
+import           Clash.Annotations.BitRepresentation.Internal (CustomReprs)
+import           Clash.Core.Var (Id)
+import           Clash.Core.TyCon (TyConMap,TyConName)
+import           Clash.Driver.Types (BindingMap)
+import           Data.IntMap.Strict           (IntMap)
```

Comment:
Weird bit of indentation here

Reply:
copy-pasta from different places
Fixed now